e2e/playwright/command-bar-tests.spec.ts

```diff
@@ -659,6 +727,8 @@ theta = 45deg
     )
   })
 
+  // TODO: add pw test for clear non-required arg button
```

Comment:
@pierremtb I think you've done this above and the `TODO` can be removed.

Reply:
https://github.com/KittyCAD/modeling-app/pull/7814/files you're right ty!